src/main/resources/templates/template_hmpps-non-associations-api.mustache

```diff
@@ -23,7 +9,7 @@
             <td class="data-column-25">Closed count</td>
         </tr>
         <tr>
-            <td>{{ optionalValue prisonName }}</td>
+            <td>{{ getPrisonName prisonId }}</td>
```

Comment:
Does this need to be swapped to prisonId if we already have the name supplied in the json

Reply:
From the ticket comments it seems like we'd prefer to use the prison id and convert ourselves, presumably so the names are displayed consistently? Whether worth having a chat with Deborah to discuss?